src/Components/Publishing/Sections/StyledText.tsx

```diff
@@ -186,5 +219,5 @@ export const StyledText = div`
 `
 
 StyledText.defaultProps = {
-  color: 'black'
+  color: "black",
 }
```

Comment:
Curious, did you find this hard to trace through and know where to add styles? I wonder if this will become increasingly difficult to manage as we add more layouts and cover more edge cases. Seems like separating them eventually will be nice.

Reply:
Most of the changes here were prettier -- was actually easier than expected to change (was only 2 lines to support news).  There are other components like 'Header' that might want to be broken out more as we add more layouts.

Reply:
True, i guess the styles are similar enough still that it doesn’t make sense yet to split out. Header for sure 👍🏼

Reply:
The one that requires the most independent styles is 'classic' layout, so as formats grow we might consider using `StyledText` as a default, and extending it `ClassicStyledText` etc. rather than using conditionals in this file.